src/core/shape_inference/include/concat_shape_inference.hpp

```diff
@@ -17,7 +17,7 @@ std::vector<TRShape> shape_infer(const Concat* op, const std::vector<T>& input_s
     NODE_VALIDATION_CHECK(op, !input_shapes.empty());
     using DimType = typename T::value_type;
 
-    auto concat_axis = op->get_axis();
+    size_t concat_axis{0};
```

Comment:
are we sure that it is always zero?

Reply:
In fact it could be not initialized here. The `concat_axis` will be normalized before use so will have correct value or error.
Initialized because not sure if Coverity will understand logic and could report error of using not initialized value.